src/layer.cpp

```diff
@@ -22,6 +22,7 @@ class mc_Scene : public cocos2d::LayerColor {
 	cocos2d::Scene *scene;
 	VALUE obj;
 	SEL update_sel;
+    cocos2d::EventListenerTouchOneByOne *touch_listener;
 
     mc_Scene() {
 	obj = Qnil;
```

Comment:
We should probably set `touch_listener` to `NULL` here (in the `mc_Scene` constructor).